var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -91,7 +99,8 @@ def cmake_args(self):
 
             # logging support
             if "+logging" in spec:
-                if spec.satisfies('@:1.2.0'):
+                if spec.satisfies('@:1.2.0') and \
+                        spec['vtk-m'].version.string != 'master':
```

Comment:
Now that #1983 has been merged, this line should no longer be necessary. `master` will never satisfy `:1.2.0`

Reply:
thanks, I'll pull these out